### DISCUSSION TITLE
Change the horizontal orientation of the camera?

### DISCUSSION THREAD

**Discussion by Kevinstorm9** [hiukim/mind-ar-js#159] [Q&A]
I am looking for some line of code to be able to change the horizontal orientation of the camera and be able to obtain a mirror effect like when you take a selfie

**Comment by eaallen** [comment]
@Aguacateferoz Here is a stack overflow that answers your question. 

[https://stackoverflow.com/questions/47742208/horizontally-flipping-getusermedias-webcam-image-stream](url)

if you want a quick fix that does not need to work with firefox, i suggest using css:
```
video {
  -webkit-transform: scaleX(-1);
  transform: scaleX(-1);
}
```
